opendebates/models.py

```diff
@@ -50,6 +50,10 @@ def user_display_name(self):
 
     editors_pick = models.BooleanField(default=False)
     approved = models.BooleanField(default=False, db_index=True)
+
+    # if True, will not show up again in moderation list.
+    moderated_removal = models.BooleanField(default=False, db_index=True)
```

Comment:
Yes, we had discussed that if an admin 'rejected' a removal-flag, then it would no longer show up in their queue. Once a question has been reviewed by an admin and felt to be appropriate, I wouldn't think they would want to keep seeing that same question over and over again (especially since questions can't be edited).